docs/package_docs.rst

```diff
@@ -86,52 +86,101 @@ We suggest this template:
-
-The canonical source of this file's content is https://www.lsstcorp.org/LegalNotices/LsstLicenseStatement.txt, though ensure that it is named ``LICENSE`` in the repository.
-
 .. _pkg-doc-copyright:
 
 COPYRIGHT
 ---------
 
-We assert copyright information in a centralized ``COPYRIGHT`` file, located in the repository's root.
+We assert copyright information in a centralized ``COPYRIGHT`` file, located in the repository's root rather than attempting to track copyright in each file separately within a package.
 Using a ``COPYRIGHT`` file allows us to maintain copyright information more effectively than in source code comments.
 (See `RFC-45 <https://jira.lsstcorp.org/browse/RFC-45>`_ for background on this policy).
 
-Default Copyright
-^^^^^^^^^^^^^^^^^
+Some code written before construction officially started will have an LSST Corporation copyright and this should not be changed unless it has erroneously been used for code after construction began in August 2015.
+During LSST construction, copyright is asserted by your employer and not centrally by AURA.
 
-By default, the ``COPYRIGHT`` file should look like:
+The ``COPYRIGHT`` file should include a line for each legal entity that contributed to the package.
+A common example may look like:
 
 .. code-block:: text
 
-   Copyright AURA/LSST (2012-2015)
+   Copyright 2012-2015 LSST Corporation
+   Copyright 2015-2017 Association of Universities for Research in Astronomy
 
 Where the year range is changed as appropriate.
+If the code in a package has not been touched since 2015 and you are working on it in 2018, do not say "2012-2018" but instead use "2012-2015, 2018".
+You can determine this by looking at the repository change history.
 
-Complex Copyright Assignments
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note::
+   Previously it was common to see AURA/LSST in the copyright assignment line.
+   This is not a legal entity and the fully-expanded name of AURA should be used.
 
-If multiple institutions contributed to the code over the same period, each institution can be listed. E.g.:
+Use a line per institution even if the years are the same:
+E.g.:
 
 .. code-block:: text
 
-   Copyright University of Washington and AURA/LSST (2012-2015)
+   Copyright 2012-2015 LSST Corporation
+   Copyright 2012-2015 University of Washington
+   Copyright 2015-2018 Association of Universities for Research in Astronomy
+
+These ``COPYRIGHT`` files should be updated during development and may be robotically refreshed.
+Automatically updating the files requires people committing to the repository use their :ref:`institutional email address <git-setup-institutional-email>`.
+
+The downside of using a single ``COPYRIGHT`` file is that moving a file between packages requires that both the source and destination ``COPYRIGHT`` files may need to be amended, since the copyright information is not included in the file being moved.
```

Comment:
Instead of starting the sentence with "The downside [..]", which makes it sound like an editorial comment, let's try to start with a task prompt. For example:

```
If you move source files between packages, remember to audit the ``COPYRIGHT`` file and move copyright assignments between packages as necessary.
```